Description: Correct pod NAME issue in LedgerSMB/Company_Config.pm
 Missing the dash in the NAME entry between the module name and the short
 description.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472544&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Company_Config.pm
+++ b/LedgerSMB/Company_Config.pm
@@ -1,6 +1,6 @@
 =head1 NAME
 
-LedgerSMB::Company_Config    Company-specific Configuration for LedgerSMB
+LedgerSMB::Company_Config - Company-specific Configuration for LedgerSMB
 
 =head1 SYNOPSIS
 
